p/scripts/main.js

```diff
@@ -1556,6 +1563,7 @@ function init_beforeDOM() {
 }
 
 function init_afterDOM() {
+	removeFirstLoadSpinner();
```

Comment:
Should not be needed for our core themes, but might be needed for other themes or extensions.